plugins/filters/ndmean/test_ndmean_mean.c

```diff
@@ -77,7 +77,7 @@ static int test_ndmean(blosc2_schunk *schunk) {
   free(smeta);
 
   if (ndim != 1) {
-    fprintf(stderr, "This test only works for ndim = 1");
+    BLOSC_TRACE_ERROR("This test only works for ndim = 1");
```

Comment:
This is a test, and I prefer the regular `printf`.  Can you remove this one?

Reply:
Done.

Together with `BLOSC_TRACE_ERROR()` everywhere in library code and `fprintf(stderr)` everywhere in examples and tests.